
FAIL event.intercept() throws if used on an event from a detached iframe assert_throws_dom: function "() => e.intercept()" did not throw

